### ISSUE TITLE
Unread counters being reset

### ISSUE THREAD

**Issue by TheSonAlsoRises** [thelounge/lounge#718]
Weird one that I may not be explaining very clearly.

Every time I click on the top channel of the first server, it resets the unread counters of every other channel, on all servers.

The actual channel being clicked on does not matter, all that is required is for it to be at the top of the list.

The counter for a given channel is restored to its original value and incremented normally only after a new message has been sent to that channel.

Running The Lounge from source (5ff1496).

**Comment by xPaw** [comment]
A couple of other users mentioned this, seems like the multi client support broke some client side selector which resets counters.

**Comment by stepie22** [comment]
This can now be closed, since PR #720 fixed this and has been merged into master.
